### ISSUE TITLE
Modernize Modern color scheme as the default scheme for all new sites

### ISSUE THREAD

**Issue by fushar** [Automattic/wp-calypso#95334]
See P2: pfsHM7-1qe-p2

We'd like to get rid the green color from the Modern admin color scheme, and use that scheme for all new sites going forward consistently.

- ~~PR to update the color scheme in Jetpack https://github.com/Automattic/jetpack/pull/39748~~
   - Most likely we won't override in dotcom and wait for Core.
- [ ] PR to update the color scheme in Calypso https://github.com/Automattic/wp-calypso/pull/95340
- [x] Patch in core https://core.trac.wordpress.org/ticket/62219

**Comment by fushar** [comment]
The patch in Core has been merged, and will be released in 6.8. After that, we should reopen the Calypso PR above and fix it on Calypso side.

**Comment by taipeicoder** [comment]
@lsl @arthur791004 I remember you recently submitted a change related to this for Atomic sites? Can this issue be closed?

**Comment by lsl** [comment]
Yes, Arthur patched wp-admin via mu plugin here: https://github.com/Automattic/jetpack/pull/42462
